rep-2000.rst

```diff
@@ -460,8 +465,9 @@ Middleware Implementation Support:
 |  rmw_connext_cpp         | RTI Connext         | Tier 1        | All Platforms except Debian | All Architectures except arm64/arm32 |
 |                          |                     |               | and OpenEmbedded            |                                      |
 +--------------------------+---------------------+---------------+-----------------------------+--------------------------------------+
-| rmw_opensplice_cpp       | ADLink OpenSplice   | Tier 2        | All Platforms except Debian | All Architectures                    |
-|                          |                     |               | and OpenEmbedded            |                                      |
+| rmw_cyclonedds_cpp       | Eclipse Cyclone DDS | Tier 2        | All Platforms               | All Architectures                    |
++--------------------------+---------------------+---------------+-----------------------------+--------------------------------------+
+| rmw_opensplice_cpp       | ADLINK OpenSplice   | Tier 2        | All Platforms except        | All Architectures                 |                          |                     |               | OpenEmbedded                |                                      |
```

Comment:
Why was the exception for Debian removed from this line?

Reply:
@dirk-thomas because we support OpenSplice on Debian. We use OpenSplice on Debian (stretch) all the time including in ADLINK AI vision, machine condition monitoring and industrial controller products. Or is there some reason I didn't know about that it had an exception listed?

Reply:
While we do have a `.deb` for Ubuntu Bionic in the apt repo there is none for Debian Buster. Therefore I don't think it should be added.

While we have built that `.deb` for Ubuntu I would rather not have us add building for more platforms. The alternative would be that you could provide `.deb` packages in the release page of OpenSplice we which just consume.

Reply:
ah, ok. I put "except Debian" back in